packages/pipeline/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig",
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
+    "lib": ["es2015", "dom"],
+    "outDir": "lib",
+    "noImplicitAny": true,
+    "strictNullChecks": true
```

Comment:
`strictNullChecks` are part of the `strict` config that is set by default in a base file
https://www.typescriptlang.org/docs/handbook/compiler-options.html